source/catalog.html

```diff
@@ -155,9 +156,11 @@ <h2 class="visually-hidden">
         </p>
       </header>
       <figure class="video__clip">
-        <source media="(min-width: 1200px)" srcset="img/video-mobile-desktop.jpg 1x, img/video-mobile-desktop@2x.jpg 2x">
-        <source media="(min-width: 768px)" srcset="img/video-mobile-tablet.jpg 1x, img/video-mobile-tablet@2x.jpg 2x">
-        <img class="video__clip-image" src="img/video-mobile.jpg" width="280" height="170" alt="Видеопрезентация">
+        <picture>
+          <source media="(min-width: 1200px)" srcset="img/video-desktop.jpg 1x, img/video-desktop@2x.jpg 2x">
+          <source media="(min-width: 768px)" srcset="img/video-tablet.jpg 1x, img/video-tablet@2x.jpg 2x">
+          <img class="video__clip-image" src="img/video-mobile.jpg 1x" width="280" height="170" alt="Видеопрезентация">
```

Comment:
Здесь аналогично главной проблемы с 1x:
```suggestion
          <img class="video__clip-image" src="img/video-mobile.jpg" width="280" height="170" alt="Видеопрезентация">
```